README-source.md

```diff
@@ -1179,7 +1179,7 @@ This makes it pretty straightforward to integrate into your testing interface. I
 ```yaml
 language: node_js
 node_js:
-  - "6.10.2"
+  - `LAMBDA_VERSION`
```

Comment:
This way? @xavdid

Reply:
oh sorry, I totally spaced on this. it would be "LAMBDA_VERSION" since I believe it's just a string replace.